features/steps/cli_steps.py

```diff
@@ -407,18 +407,17 @@ def update_pyproject_toml(context: behave.runner.Context, new_source_dir):
 
 @given("I have updated kedro requirements")
 def update_kedro_req(context: behave.runner.Context):
-    """Replace kedro as a standalone requirement with a line
-    that includes all of kedro's dependencies (-r kedro/requirements.txt)
+    """Remove kedro as a standalone requirement.
     """
     reqs_path = context.root_project_dir / "src" / "requirements.txt"
-    kedro_reqs = f"-r {context.requirements_path.as_posix()}"
 
     if reqs_path.is_file():
         old_reqs = reqs_path.read_text().splitlines()
         new_reqs = []
         for req in old_reqs:
             if req.startswith("kedro"):
```

Comment:
I'm not even sure this was correct to start with, now that we have `kedro-datasets`, `kedro-airflow` etc. I should probably parse the requirement and only remove `kedro` proper.